app/src/main/java/com/github/multimatum_team/multimatum/repository/FirebasePdfRepository.kt

```diff
@@ -20,9 +22,10 @@ class FirebasePdfRepository @Inject constructor(database: FirebaseStorage) : Pdf
         if (data != Uri.EMPTY) {
             val ref =
                 storageReference.child(
-                    FirebaseAuth.getInstance().uid + "/upload" + (Int.MIN_VALUE..Int.MAX_VALUE).random()
-                        .toString() + PDFUtil.getFileNameFromUri(
-                        data
+                    FirebaseAuth.getInstance().uid + "/" + PDFUtil.addRdmCharToStr(
```

Comment:
Please do `@Inject constructor(auth: FirebaseAuth, database: FirebaseStorage)` instead of getting the instance, this way we can test it by replacing the auth by a mocked object